hack/run-compose.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# This script builds the schema, then tears down and rebuilds
+# the local podman-compose development environment.
+#
+# It is intended to be run from the root of the project.
+
+set -e
+
+# Path to compose file
+COMPOSE_FILE="./podman/podman-compose.yml"
+
+echo "--- Building the component with schemas ---"
+bash ./hack/build-with-schemas.sh
+
+echo "--- Switching to podman folder ---"
+pushd podman
+
+echo "--- Stopping existing podman-compose ---"
+podman-compose down
+
+echo "--- Starting podman-compose ---"
+podman-compose up --build
```

Comment:
I get permission issues here from the kafka container, I fixed by adding
```
diff --git a/hack/run-compose.sh b/hack/run-compose.sh
index 8d9878c..fa5d6a4 100755
--- a/hack/run-compose.sh
+++ b/hack/run-compose.sh
@@ -16,6 +16,10 @@ bash ./hack/build-with-schemas.sh
 echo "--- Switching to podman folder ---"
 pushd podman
 
+echo "--- Creating bindmounts --"
+[ ! -d "./kafka-data" ] && mkdir ./kafka-data && podman unshare chown 1001:0 ./kafka-data
+[ ! -d "./kafka-config" ] && mkdir ./kafka-config && podman unshare chown 1001:0 ./kafka-config
```

Reply:
Done